games-engines/fheroes2/fheroes2-1.0.4.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake desktop
```

Comment:
Does this actually use the desktop eclass? It should use the xdg eclass though to update the icon cache.

Reply:
Done